src/wallet/rpcdump.cpp

```diff
@@ -614,11 +611,10 @@ UniValue importwallet(const JSONRPCRequest& request)
                     }
                 }
                 pwallet->WalletLogPrintf("Importing %s...\n", EncodeDestination(keyid));
-                if (!pwallet->AddKeyPubKey(key, pubkey)) {
+                if (!pwallet->AddKeyPubKey(key, pubkey, nTime)) {
                     fGood = false;
                     continue;
                 }
-                pwallet->mapKeyMetadata[keyid].nCreateTime = nTime;
                 if (fLabel)
                     pwallet->SetAddressBook(keyid, strLabel, "receive");
                 nTimeBegin = std::min(nTimeBegin, nTime);
```

Comment:
I think this line no longer does anything and can be dropped. If `nTime` is less than `nTimeBegin`, then the `UpdateTimeFirstKey` call inside `AddKeyPubKey` will have already updated the minimum time.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15032#discussion_r244844275

> nTimeBegin is also used below in the RescanWallet() call, so I think it needs to be set to the min value here.

You're right. Not sure how I missed the RescanWallet call.